ADAL/src/cache/ADTokenCacheAccessor.m

```diff
@@ -91,10 +94,20 @@ - (ADTokenCacheItem *)getItemForUser:(ADUserIdentifier *)identifier
 - (ADTokenCacheItem *)getATRTItemForUser:(ADUserIdentifier *)identifier
                                 resource:(NSString *)resource
                                 clientId:(NSString *)clientId
-                           correlationId:(NSUUID *)correlationId
+                           requestParams:(ADRequestParameters*)requestParams
                                    error:(ADAuthenticationError * __autoreleasing *)error
 {
-    return [self getItemForUser:identifier resource:resource clientId:clientId correlationId:correlationId error:error];
+    [[ADTelemetry sharedInstance] startEvent:[requestParams telemetryRequestId] eventName:@"token_cache_lookup"];
+    
+    ADTokenCacheItem* item = [self getItemForUser:identifier resource:resource clientId:clientId requestParams:requestParams error:error];
+    
+    ADTelemetryCacheEvent* event = [[ADTelemetryCacheEvent alloc] initWithName:@"token_cache_lookup"];
+    [event setTokenType:@"access token/refresh token"];
```

Comment:
we just need to know whether the refresh token is FRT/MRRT/...? Maybe we need to start this event one level up.